test/components/contextMenuTest.js

```diff
@@ -0,0 +1,106 @@
+/* global describe, it, before */
+
+const Brave = require('../lib/brave')
+const {urlInput} = require('../lib/selectors')
+
+describe('ContextMenu', function () {
+  function * setup (client) {
+    yield client
+      .waitForUrl(Brave.newTabUrl)
+      .waitForBrowserWindow()
+      .waitForEnabled(urlInput)
+  }
+
+  describe('navigation', function () {
+    Brave.beforeAll(this)
+
+    before(function *() {
+      yield setup(this.app.client)
+      this.formfill = Brave.server.url('formfill.html')
+      this.input = '[name="01___title"]'
+      this.values = ['Value 1', 'Value 2', 'Value 3', 'Value 4']
+
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+
+      for (let i = 0; i < this.values.length; i++) {
+        yield this.app.client
+          .click(this.input)
+          .keys(this.values[i])
+          .click('#submit')
+          .waitForVisible('<form>')
+      }
+    })
+
+    it('check if context menu exists', function *() {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+        .click(this.input)
+        .click(this.input)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForVisible('.contextMenuItemText')
+    })
+
+    it('click on item', function *() {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+        .click(this.input)
+        .click(this.input)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForVisible('.contextMenu')
+        .click('.contextMenuItem')
+        .tabByUrl(this.formfill)
+        .getValue(this.input).should.eventually.be.equal(this.values[0])
+    })
+
+    it('select item via click and keys', function *() {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+        .click(this.input)
+        .click(this.input)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForVisible('.contextMenu')
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE007')
+        .pause(10)
+        .tabByUrl(this.formfill)
+        .getValue(this.input).should.eventually.be.equal(this.values[1])
+    })
+
+    it('select item via keys only', function *() {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+        .click(this.input)
+        .windowByUrl(Brave.browserWindowUrl)
+        .keys('\uE015')
+        .waitForVisible('.contextMenu')
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE013')
+        .pause(10)
+        .keys('\uE007')
```

Comment:
Currently some test's are failing, still investigating why this is happening. If I do test manually (added longer pause) with my mouse and keyboard, test is successful.